config/wpe_config.toml

```diff
@@ -78,4 +78,4 @@ enclave_type = "WPE"
 [KMEListener]
 # URL of KMEListener where a Work order processor can send requests meant for
 # the Key Management Enclave(KME)  
-bind = "http://avalon-kme:1948"
+kme_listener_url = "http://avalon-kme:1948"
```

Comment:
Please add command argument to WPE manager as well. If command line argument is passed it takes preference. In docker mode we pass argument and in config file by default it will be localhost.. This way we can handle both stand alone and docker based builds..

Reply:
I prefer that convention too--localhost as the default and overridden in Docker.

Reply:
Updated @danintel  @Ram-srini .